pkg/helm/chart/templates/Deployment.apps/bootstrap-autoapprover.yaml

```diff
@@ -71,6 +71,10 @@ spec:
         - name: kubeconfig
           mountPath: /tmp/kubeconfig
           readOnly: true
+{{if ne "" .Values.ContainerPullSecret }}
```

Comment:
would it be better to do this via https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account ? I suspect it might be?

Reply:
We run all HCP via default service account. This means secret adding will be `kubectl patch` command in create.sh script. If this is sufficient (considering we will do this in dev only), yes its easier. Otherwise, we either need to move to other SA and create it as part of helm install.